holopy/inference/interface.py

```diff
@@ -74,36 +74,59 @@ def validate_strategy(strategy, operation):
     return strategy
 
 
-def make_default_model(base_scatterer, fitting_parameters):
+def make_default_model(base_scatterer, fitting_parameters=None):
     if fitting_parameters is None:
-        fitting_parameters = base_scatterer.parameters.keys()
+        fitting_parameters = list(base_scatterer.parameters.keys())
```

Comment:
Did you find this because a test failed? If so, it might be a sign that we need to test this better. (Unless if the parameters don't actually need to be a list.)

Reply:
Turned out the list casting wasn't actually necessary here, so I've reverted the change.